codespell_lib/data/dictionary.txt

```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
 aautomatically->automatically
 abailable->available
 abanden->abandon
+abandenment->abandonment
+abandining->abandoning
+abandomnent->abandonment
+abandond->abandoned
 abandonded->abandoned
+abandonding->abandoning
 abandone->abandon
+abandonig->abandoning
+abandonne->abandonment, abandon,
 abandonned->abandoned
+abandonnent->abandonment
 abandonning->abandoning
+abanonded->abandoned
```

Comment:
Perhaps also:
```
abanond->abandon
abanonding->abandoning
abanonds->abandons
```
I have not found `abanonding` in GitHub, only in the internet at large with Google.